core/src/types/token.rs

```diff
@@ -1,6 +1,7 @@
 //! A basic fungible token
 
 use std::cmp::Ordering;
+use std::collections::BTreeMap;
```

Comment:
How do we decide what gets its own module and what stays under core? For example, why isn't this its own module or account not under core?

Reply:
we should probably move this into the new trans_token crate - there are still few files left in core that depend on it, but they can also be moved out